test/e2e/policies.test-suite/balloons/n4c16/test06-update-configmap/code.var.sh

```diff
@@ -15,6 +15,10 @@ cleanup() {
 	kubectl -n kube-system delete configmap nri-resmgr-config.default || :"
     vm-port-forward-disable
     terminate nri-resmgr
+
+    # Just in case the cache says that the policy is "topology-aware"
+    # (from earlier tests) then remove the cache to force "balloons" policy
+    vm-command "rm -f /var/lib/nri-resmgr/cache" || true
```

Comment:
I think this is fine for the tests. That said, I think we should consider changing our startup logic that
- we fail if a particular policy is explicitly asked for in the configuration and we're not that policy
- in all other cases if our policy does not match the cached one, we reset the policy (and its data) from the cache
IOW, if one does not specify in the configuration a policy explicitly, then we consider the binary being started as the primary means of selecting the policy. Or to put it in another way, if the cached policy does match the one in the binary, we implicitly (agree to) switch the policy on startup instead of failing.